.github/workflows/tests.yml

```diff
@@ -73,7 +73,7 @@ jobs:
     timeout-minutes: 25
     strategy:
       matrix:
-        tarantool-version: [ "1.x-centos7", "2.11.2-centos7", "3.0.1" ]
+        tarantool-version: [ "1.x-centos7", "2.11.2-ubuntu20.04", "3.0.1" ]
```

Comment:
Why we don't have the same problem with 1.x-centos7?

Reply:
Because it is `TarantoolContainer`, not `TarantoolCatridgeContainer`